README.md

```diff
@@ -55,10 +55,10 @@ customVcc.save();
 
 ### Using custom controls
 
-To make a custom control accessible to Koji templates, you must serve it at a custom domain.
+To make a custom control accessible to Koji app, you must serve it at a custom domain.
 After you publish the project, add a subdomain under the `koji-vccs.com` root domain.
 
-Then, you can use the control in a Koji template in one of these ways.
+Then, you can use the control in a Koji app in one of these ways.
 
 * (Recommended) With the @withkoji/core package, call the `Koji.ui.capture.custom` method with the subdomain name. For example:
 
```

Comment:
Updates needed to related resources: 
- Update link and title of "Building your first custom VCC" to point to the current page ("Building custom controls")
- Remove Custom VCC blueprint (that page has been deprecated)

Reply:
Fixed.